### PR TITLE
Changed api logging feature - docs

### PR DESCRIPTION
**Pull request by amulyavarote**
Signed-off-by: Amulya Varote <amulyavarote@microsoft.com>

Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [x] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [x] Commands include options for Linux, MacOS, and Windows within codetabs
- [x] New file and folder names are globally unique
- [x] Page references use shortcodes instead of markdown or URL links
- [x] Images use HTML style and have alternative text
- [x] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Changed api logging feature from api-log-level to enable-api-logging flag.
NA

### CODE REVIEW

File: daprdocs/content/en/operations/troubleshooting/api-logs-troubleshooting.md [hunk 1]
```diff
@@ -6,42 +6,36 @@ weight: 3000
...
+To enable API logging, you can use the `--enable-api-logging` command-line option. For example:
 
 ```bash
-./daprd --api-log-level info
-./daprd --api-log-level debug
+./daprd --enable-api-logging
 ```
 
-This starts the Dapr runtime with a api log level of `info` and `debug` accordingly.
+This starts the Dapr runtime with a api log level of `info`.
```
Comment (msfussell) [inline daprdocs/content/en/operations/troubleshooting/api-logs-troubleshooting.md]:
```suggestion
This starts the Dapr runtime with API logging.
```

File: daprdocs/content/en/operations/troubleshooting/api-logs-troubleshooting.md [hunk 2]
```diff
@@ -6,42 +6,36 @@ weight: 3000
...
-dapr run --api-log-level info node myapp.js
+dapr run --enable-api-logging node myapp.js
 ```
 
 ### Viewing API logs in self hosted mode
 
 When running Dapr with the Dapr CLI, both your app's log output and the Dapr runtime log output are redirected to the same session, for easy debugging.
 
-The below example shows `info` level API logging:
+The below example shows API logs:
```
Comment (msfussell) [inline daprdocs/content/en/operations/troubleshooting/api-logs-troubleshooting.md]:
```suggestion
The example below shows some API logs:
```

File: daprdocs/content/en/reference/cli/dapr-run.md [hunk 3]
```diff
@@ -36,7 +36,7 @@ dapr run [flags] [command]
 | `--help`, `-h`                 |                      |                                                                                    | Print this help message                                                                              |
 | `--image`                      |                      |                                                                                    | The image to build the code in. Input is: `repository/image`                                         |
 | `--log-level`                  |                      | `info`                                                                             | The log verbosity. Valid values are: `debug`, `info`, `warn`, `error`, `fatal`, or `panic`           |
-| `--api-log-level`                  |                      | `debug`                                                                             | The API log verbosity. Valid values are: `debug` or `info`        |
+| `--enable-api-logging`                  |                      | `false`                                                                             | The API log verbosity.        |
```
Comment (msfussell) [inline daprdocs/content/en/reference/cli/dapr-run.md]:
```suggestion
| `--enable-api-logging`                  |                      | `false`                                                                             | Enable the logging of all API calls from application to Dapr      |
```

Comment (msfussell) [reply daprdocs/content/en/reference/cli/dapr-run.md]: Can you add an example in the examples sections in the file. Like this


```
# Run a NodeJs application that listens to port 3000 with API logging enabled
dapr run --app-id myapp --app-port 3000 --enable-api-logging  -- node myapp.js
```

File: daprdocs/content/en/operations/troubleshooting/api-logs-troubleshooting.md [hunk 4]
```diff
@@ -6,42 +6,36 @@ weight: 3000
...
 
 ```bash
-dapr run --api-log-level info node myapp.js
+dapr run --enable-api-logging node myapp.js
```
Comment (msfussell) [inline daprdocs/content/en/operations/troubleshooting/api-logs-troubleshooting.md]:
```suggestion
dapr run --enable-api-logging -- node myapp.js
```

**Review by msfussell — COMMENTED** [review]
Added a few comments to look at.

**Comment by msfussell** [comment]
I just realize that we have to tie this into this article https://docs.dapr.io/operations/monitoring/logging/logs/ needs to mention API logs as well as have hyper-links to both logTrouble shooting articles. 

Can you add these links at the end 
[Configure and view Dapr Logs]({{< ref "logs-troubleshooting.md" >}}) 
[Configure and view Dapr API Logs]({{< ref "api-logs-troubleshooting.md" >}})

And add this heading and paragraph just before the Log Collectors heading

*** API Logging ***
API logging enables you to see the API calls from your application to the Dapr sidecar to debug issues. You can combine both Dapr API logging with Dapr log events. See [configure and view Dapr Logs]({{< ref "logs-troubleshooting.md" >}}) and [configure and view Dapr API Logs]({{< ref "api-logs-troubleshooting.md" >}}) for more information.

**Comment by msfussell** [comment]
Also thought of a question. When setting --log-as-json, will this cause the API logs to be written as JSON? It should IMO. See https://docs.dapr.io/operations/monitoring/logging/logs/

**Comment by artursouza** [comment]
> Also thought of a question. When setting --log-as-json, will this cause the API logs to be written as JSON? It should IMO. See https://docs.dapr.io/operations/monitoring/logging/logs/

It uses the same logger class as the rest of the code.

**Review by msfussell — APPROVED** [review]
Wonderful!

**Pull request merged**